src/main/java/flaxbeard/steamcraft/block/BlockSteamCharger.java

```diff
@@ -65,26 +80,12 @@ public boolean onBlockActivated(World world, int x, int y, int z, EntityPlayer p
             tile.setInventorySlotContents(0, null);
         } else {
             if (player.getHeldItem() != null) {
-                if (player.getHeldItem().getItem() instanceof ISteamChargable) {
-                    ISteamChargable item = (ISteamChargable) player.getHeldItem().getItem();
-                    if (item.canCharge(player.getHeldItem())) {
-                        ItemStack copy = player.getCurrentEquippedItem().copy();
-                        copy.stackSize = 1;
-                        tile.setInventorySlotContents(0, copy);
-                        player.getCurrentEquippedItem().stackSize -= 1;
-                        tile.randomDegrees = world.rand.nextInt(361);
-                    }
-                } else if (CrossMod.TINKERS_CONSTRUCT
```

Comment:
Hm, looks like I messed up when merging this file. I need to re-add the TiCo stuff.